frontstage/controllers/iac_controller.py

```diff
@@ -21,18 +21,16 @@ def get_iac_from_enrolment(enrolment_code, validate=False):
     except requests.exceptions.HTTPError:
         if response.status_code == 404:
             logger.info('IAC not found')
-            if validate:
-                raise ApiError(logger, response, message='Invalid enrolment code used')
             return
         # 401s may include error context in the JSON response
         elif response.status_code != 401:
             raise ApiError(logger, response, message='Failed to retrieve IAC')
 
-    if validate and not response.json().get('active', False):
-        raise ApiError(logger, response, message='Invalid enrolment code used')
+    if response.json().get('active', False):
+        logger.info("Invalid IAC used")
```

Comment:
status code?

Reply:
Does this need to return False/None or something other than the JSON response below to show it's inactive?

Reply:
The status code won't be useful here as it will be 200

Reply:
not from the Api?

Reply:
Yeah the api returns 200 as it successfully got the iac code it's just not active